components/events/SpeechCard.i18n.js

```diff
@@ -2,6 +2,11 @@ import { genI18nMessages } from '~/utils/i18n.utils'
 
 export default genI18nMessages({
     'en-us': {
+        levels: {
+            EXPERIENCED: 'H',
+            INTERMEDIATE: 'M',
+            NOVICE: 'L',
```

Comment:
should follow the spec but the spec is also pretty weird.
I'll talk to the designer about this issue.

Reply:
[b4c47bb](https://github.com/pycontw/pycontw-frontend/pull/259/commits/b4c47bb54b8e2903ce43676399d7e9079cab2a57)

Reply:
e11e82f